modules/nw-openstack-external-ccm.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+// TODO: GitHub link clearance.
+
+:_content-type: CONCEPT
+[id="nw-openstack-external-ccm_{context}"]
+= The {rh-openstack} Cloud Controller Manager
+
+In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack} cloud provider to the link:https://github.com/kubernetes/cloud-provider-openstack[external Cloud Controller Manager (CCM)]. This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
```

Comment:
@sferich888 We would like to include a link to the OpenStack CCM repo in this page. Is this something that we could allow, as there isn't anywhere else to point at the moment, or would the No GitHub Links policy preclude it?

Reply:
I don't see a reasons why a link here needs to be provided; or what purpose or value it serves.

Reply:
```suggestion
In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack} cloud provider to the external OpenStack Cloud Controller Manager (CCM). This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
```